ci-operator/step-registry/tcpdump-service/tcpdump-service-commands.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+echo "Injecting manifest to deploy tcpdump service MachineConfig with script and systemd units"
+
+echo
+
+# Echo a script to be run as our systemd unit to a file so we can base64 encode it.
+sysd_script=$(cat << EOF
+#!/bin/sh
+
+set -e
+
+# Build up a tcpdump filter for all quay.io IPs. These change seemingly daily
+# so by doing the lookup before we launch we increase our chances of dumping
+# the packets we're after.
+quay_lines=\$(dig quay.io | grep "^quay.io\." | awk '{print \$5}')
+tcpdump_filter=""
+for i in \$quay_lines
+do
+    if [ -z "\$tcpdump_filter" ]
+    then
+        tcpdump_filter+="(host \$i"
+    else
+        tcpdump_filter+=" or host \$i"
+    fi
+done
+# Add in registry.ci.openshift.org IPs as of 2021-12-22 and close the filter:
```

Comment:
will this need updating?

Reply:
Still accurate today so it looks fairly stable, but it's entirely possible they would. I can repeat the above mess for registry.ci if we think this will live for awhile.

Reply:
Addressed this with a much simplified dig command that looks up A records for both, and filters out CNAME.